app/scripts/metamask-controller.js

```diff
@@ -744,10 +744,12 @@ export default class MetamaskController extends EventEmitter {
 
     // Accounts
     const hdKeyring = this.keyringController.getKeyringsByType('HD Key Tree')[0]
+    const simpleKeyPairKeyring = this.keyringController.getKeyringsByType('Simple Key Pair')[0]
     const hdAccounts = await hdKeyring.getAccounts()
+    const simpleKeyPairAccounts = await simpleKeyPairKeyring.getAccounts()
```

Comment:
It looks like this would only get the first imported account, rather than all imported accounts. It might be better to get _all_ of the keyrings, since a new one is created for each imported account.

Something like this might work for getting them all:

```suggestion
    const simpleKeyPairKeyrings = this.keyringController.getKeyringsByType('Simple Key Pair')
    const hdAccounts = await hdKeyring.getAccounts()
    const simpleKeyPairKeyringAccounts = await Promise.all(
      simpleKeyPairKeyrings.map((keyring) => keyring.getAccounts())
    )
    const simpleKeyPairAccounts = simpleKeyPairKeyringAccounts.reduce((acc, accounts) => [...acc, ...accounts], [])
```

Reply:
this is very true, thanks @Gudahtt !